<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectSC</class>
 <widget class="QDialog" name="SelectSC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор доступной СК</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="createSC_pb">
         <property name="text">
          <string>Создать СК</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_pb">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editSC_pb">
         <property name="text">
          <string>Изменить СК</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setSC_pb">
         <property name="text">
          <string>Выбрать СК</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
